scripts/common_startup.sh

```diff
@@ -9,6 +9,7 @@ done
 # Conda Python is in use, do not use virtualenv
 if python -V 2>&1 | grep -q -e 'Anaconda' -e 'Continuum Analytics' ; then
     SET_VENV=0
+    CONDA_ALREADY_INSTALLED=1
```

Comment:
I am actually agreeing with the way you wrote it originally, sorry that was not clear.  When I replied to the comment I thought it was attached to that other block :)